EppoEngine/Source/Renderer/Mesh/Material.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "Asset/Asset.h"
+
+namespace Eppo
+{
+	class Material : public Asset
+	{
+	public:
+		Material() = default;
+
+		std::string m_Name;
+
+		glm::vec3 m_AmbientColor;
+		glm::vec3 m_DiffuseColor;
+		glm::vec3 m_SpecularColor;
```

Comment:
Shaders are in "EppoEditor/Resources/Shaders"

Reply:
That's because the name is slightly different.

layout(binding = 4) uniform Material
{
	vec4 AmbientColor;
	vec4 DiffuseColor;
	vec4 SpecularColor;
	float Roughness;
} uMaterial;

This is it.

Reply:
Don't do that. I use the m_ prefix in C++ for denoting member variables. Shaders are an entirely different things. Don't try to couple those together.